webapp/src/components/common/Matrix/components/MatrixStats/index.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2025, RTE (https://www.rte-france.com)
+ *
+ * See AUTHORS.txt
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This file is part of the Antares project.
+ */
+
+import { Box, Paper, Typography, Divider } from "@mui/material";
+import { formatGridNumber } from "../../shared/utils";
+
+interface MatrixStatsProps {
+  stats: {
+    count: number;
+    sum: number;
+    average: number;
+    min: number;
+    max: number;
+  } | null;
+}
+
+function MatrixStats({ stats }: MatrixStatsProps) {
+  if (!stats) {
```

Comment:
Better to add all logic here:

```typescript

interface MatrixStatsProps {
  data: number[][];
  gridSelection: GridSelection;
  gridToData: (item: Item) => Item | null;
}

function MatrixStats({ data, gridSelection, gridToData }: MatrixStatsProps) {
  const stats = useSelectionStats({
    data,
    selection: gridSelection,
    gridToData,
  });

  // ...
  
}
```

Reply:
this has pros and cons, at that moment I prefer to keep it a presentational component